### PR TITLE
ENH: add Ultrasound remote module

### PR DESCRIPTION
**Pull request by dzenanz**
## PR Checklist
- [ ] No [API changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#breaking-changes) were made (or the changes have been approved)
- [ ] No [major design changes](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CONTRIBUTING.md#design-changes) were made (or the changes have been approved)
- [x] Added test (or behavior not changed)
- [x] Updated API documentation (or API not changed)
- [x] Added [license](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/KWStyle/ITKHeader.h) to new files (if any)
- [x] Added Python wrapping to new files (if any) as described in [ITK Software Guide](https://itk.org/ItkSoftwareGuide.pdf) Section 9.5
- [x] Added [ITK examples](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples) for all new major features (if any)

### CODE REVIEW

**Comment by dzenanz** [comment]
@thewtex I think this is a good time to index it in ITK?

**Comment by thewtex** [comment]
> @thewtex I think this is a good time to index it in ITK?

Yes, I think we are there. We will likely have a few updates before the release.

File: Modules/Remote/Ultrasound.remote.cmake [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#--   - [ ] All requirements of Levels 4,3,2,1
+#--
+#-- ## Compliance Level 4 star (Very high-quality code, perhaps small community dependance)
+#--   - [X] Meets all ITK code style standards
+#--   - [ ] No external requirements beyond those needed by ITK proper
+#--   - [X] Builds and passes tests on all supported platforms within 1 month of each core tagged release
+#--            - [X] Windows Shared Library Build with Visual Studio
+#--            - [X] Mac with clang compiller
+#--            - [X] Linux with gcc compiler
+#--   - [ ] Active developer community dedicated to maintaining code-base
```
Comment (thewtex) [inline Modules/Remote/Ultrasound.remote.cmake:19]: +1 `X`

**Review by thewtex — APPROVED** [review]

File: Modules/Remote/Ultrasound.remote.cmake (hunk 1 above)
Comment (dzenanz) [reply Modules/Remote/Ultrasound.remote.cmake]: Done

**Pull request merged**